css/site.styl

```diff
@@ -354,26 +354,35 @@ img
 // ******************************************************
 
 // HOME
-
-.home-item
-  width: 50%
-  text-align: center
-  .attachment-home-thumb
-    display: block
-  a
-    display: inline-block
-    cursor: pointer 
+#home-holder 
+  pointer-events: none
+  &:hover
+    .attachment-home-thumb
+      opacity: 0
+    .home-item-image-holder
+      border: $border
+  .home-item
+    pointer-events: auto
+    width: 50%
+    .attachment-home-thumb
+      display: block
+    a
+      display: inline-block
+      cursor: pointer 
     &:hover
+      .attachment-home-thumb
+        opacity: 1
       .home-item-image-caption
         opacity: 1
-      .attachment-home-thumb
-        opacity: 0
       .home-item-image-holder
-        border: $border
+        border: 3px solid transparent
+
```

Comment:
is there not some weird css parent selector or something so the hover can be put on the image? If not I think it needs to be js as its kinda annoying that it fires when on the item padding which is quite a lot now.